ext/afform/core/ang/af/afForm.component.js

```diff
@@ -4,7 +4,7 @@
     bindings: {
       ctrl: '@'
     },
-    controller: function($scope, $routeParams, $timeout, crmApi4, crmStatus) {
+    controller: function($scope, $routeParams, $timeout, crmApi4, crmStatus, $window) {
```

Comment:
Not sure but I think the [`$location`](https://docs.angularjs.org/api/ng/service/$location) service might be a better choice?

Reply:
@colemanw not if we are redirecting somewhere else see under the What does it not do section here https://docs.angularjs.org/guide/$location

Reply:
Ok cool.